nextflow.config

```diff
@@ -278,9 +303,10 @@ plugins {
 
 validation {
     defaultIgnoreParams = ["genomes"]
+    monochromeLogs = params.monochrome_logs
     help {
         enabled = true
-        command = "nextflow run $manifest.name -profile <docker/singularity/.../institute> --input samplesheet.csv --outdir <OUTDIR>"
+        command = "nextflow run nf-core/denovotranscript -profile <docker/singularity/.../institute> --input samplesheet.csv --outdir <OUTDIR>"
```

Comment:
out of curiosity and for my understanding: why is the `manifest.name` here removed since it is still in the `manifest` section? @mirpedrol ?

Reply:
https://github.com/nf-core/tools/pull/3225

Reply:
ok thank you :)